README.md

```diff
@@ -7,6 +7,11 @@
 
 The project is a collaboration with [mgostIH](https://github.com/mgostIH), on whose idea the initial library design is based on and who continues to provide incredibly helpful insights into optimization, Geometry Dash and Rust.
 
+## Guide
+If you are planning to use this for your own demonlist that isn't related to the main Geometry Dash servers, you can use:
+`rust
+GDPS_URL.get_or_init(|| "https://your-custom-gdps-url.com".to_string());`
```

Comment:
```suggestion
```rust
REQUEST_BASE.get_or_init(|| "https://your-custom-gdps-url.com");`
\```
in your `fn main` (or anywhere before the first call to any dash-rs functions)
```

(without the backslack at the end, turns out putting backticks into codeblocks is quite hard haha)